build_tools/scripts/bazel_to_cmake.py

```diff
@@ -203,30 +203,22 @@ def _convert_tblgen_block(self, tblgen):
       return ""
 
   def _convert_target(self, target):
-    if target.startswith(":"):
-      # Bazel package-relative `:logging` -> CMake absolute `iree::base::logging`
-      package = os.path.dirname(self.converter.rel_build_file_path)
-      package = package.replace(os.path.sep, "::")
-      if package.endswith(target):
-        target = package  # Omit target if it matches the package name
-      else:
-        target = package + ":" + target
-      if target.endswith("_gen"):
-        # Files created by gentbl have to be included as source and header files
-        # and not as a dependency. Adding these targets to the dependencies list,
-        # results in linkage failures if the library including the gentbl dep is
-        # marked as ALWAYSLINK.
-        # TODO: Some targets not to be included end to "Gen", but others like
-        #       LLVMTableGen still need to be included.
-        return ""
-    elif not target.startswith("//iree"):
+    if target.startswith(":") and target.endswith(("_gen","Gen")):
```

Comment:
nit: space after comma

pylint should do this for you

http://google.github.io/styleguide/pyguide.html#21-lint
http://google.github.io/styleguide/pyguide.html#36-whitespace

Reply:
Thanks for providing the references. Pylint is not really happy with the current scripts anyway ;)

Reply:
I have my editor set up to run pylint/clang-format/etc. on file save so I usually don't have to care about indentation, spacing, trailing spaces/newlines, etc. There could be some differences between the formatting that the Google configuration of those tools uses and the global defaults though.

This case might even be handled by our import automation, but we aim to keep the merged code as close to the code in the PR as possible.

Reply:
Yes, I agree that the code should keep close. I'll will be more careful next time.